src/Core/TileMesh.js

```diff
@@ -101,6 +102,43 @@ class TileMesh extends THREE.Mesh {
             this.material.updateLayersUniforms();
         }
     }
+
+    /**
+     * Refresh a material of this tile, associated with a specific layer. Only
+     * works with ColorLayer and ElevationLayer for now. It also can't work with
+     * a layer containing images like `png` or `jpeg`, as there is no way to
+     * specify the necessary flag to retrieve images again from a server.
+     *
+     * @param {ColorLayer|ElevationLayer} layer - The layer associated to the
+     * material.
+     * @param {View} [view] - The view to update, don't forget to call
+     * `view.notifyChange` otherwise.
+     * @param {boolean} [traverse=false] - Refresh the children of the tile - if
+     * any.
+     */
+    refreshMaterial(layer, view, traverse = false) {
+        if (layer.isColorLayer || layer.isElevationLayer) {
+            const materialLayer = this.material.getLayer(layer.id);
+            if (materialLayer) {
+                materialLayer.textures.forEach(t => Cache.delete(layer.source.uid, layer.id, t.extent.toString('-')));
+            }
+
+            this.material.removeLayer(layer.id);
+            this.layerUpdateState[layer.id].forceUpdate();
+
+            if (traverse) {
+                this.traverse((c) => {
+                    if (c.isTileMesh) {
+                        c.refreshMaterial(layer);
+                    }
+                });
+            }
+        }
+
+        if (view) {
```

Comment:
could move `notifyChange` from this file?

Reply:
Removed it and specified it in the documentation that it needs to be called.